staging/src/github.com/openshift/openshift-controller-manager/pkg/build/controller/build/build_controller.go

```diff
@@ -381,7 +435,8 @@ func (bc *BuildController) Run(workers int, stopCh <-chan struct{}) {
 		bc.serviceAccountStoreSynced,
 		bc.imageStreamStoreSynced,
 		bc.openshiftConfigConfigMapStoreSynced,
-		bc.controllerManagerConfigMapStoreSynced) {
+		bc.controllerManagerConfigMapStoreSynced,
+		bc.proxyCfgStoreSynced) {
```

Comment:
This will break the integration tests - move down to the next `WaitForCacheSync` block

Reply:
ah ... that was what explained the golang thread dump I saw :-) ... thanks for the assist @adambkaplan